pagopa/pn-frontend
# PN-8619: fixed "from date" and "to date" filters in pa/pf/pg

## Short description
Fixed date filters in pa, pf and pg in notification list.

## List of changes proposed in this pull request
- Fixed "to date" filter when "to date" filter is set to today, using endOfDay instead of nextDay
- Updated "from date" filter setting the startOfDay

## How to test
Test 1. Set date filters, go in notification detail, go back and check date filters are still set.
Test 2. Set date filters where EndDate is equal to today, go in notification detail, go back and check date filters are still set.
Test 3. Do test 1 and test 2 checking in devTools that "from date" and "to date" are equal respectively to the beggining of the day (time 00:00:00) and to the end of the day (23:59:59,999)
